shared/interface/IRelativePerformance.ts

```diff
@@ -0,0 +1,49 @@
+import {UnitOfMeasurementType} from './IUnitOfMeasurement';
+import {IRelative} from './Problem/IProblemRelativePerformance';
+
+type TRelativePerformanceType =
+  | 'relative-identity-normal'
+  | 'relative-logit-normal'
+  | 'relative-log-normal'
+  | 'relative-cloglog-normal'
+  | 'relative-smd-normal'
+  | 'relative-normal'
+  | 'relative-survival';
+
+type TBaseline =
+  | 'dnorm'
+  | 'dsurv'
+  | 'dbeta-logit'
+  | 'dbeta-cloglog'
+  | 'dt'
+  | 'identity'
+  | 'logit'
+  | 'log'
+  | 'cloglog'
+  | 'smd';
+
+export default interface IRelativePerformance {
+  dataSourceId: string;
+  criterionId: string;
+  unitOfMeasurementType?: UnitOfMeasurementType;
+  type: TRelativePerformanceType;
+  baseline: {
```

Comment:
Should we put in a TODO comment for splitting this out into separate relative performances, or will we find this when it becomes relevant?